explorer/client/src/components/Admin/Operators/Operator.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { KeyValueList } from '../../KeyValueList'
+import { OperatorShowData } from '../../../reducers/adminOperatorsShow'
+
+interface Props {
+  operatorData: OperatorShowData
+  loadingMessage: string
+}
+
+const entries = (operatorData: OperatorShowData): [string, string][] => {
+  return [
+    ['url', operatorData.url || ''],
+    ['uptime', operatorData.uptime.toString()],
+    ['job runs completed', operatorData.jobCounts.completed.toString()],
+    ['job runs errored', operatorData.jobCounts.errored.toString()],
+    ['job runs in progress', operatorData.jobCounts.inProgress.toString()],
+    ['total job runs', operatorData.jobCounts.total.toString()],
+  ]
+}
+
+const Operator: React.FC<Props> = ({ operatorData, loadingMessage }) => {
```

Comment:
Looks like loadingMessage can be removed from props and be defined within this file

Reply:
Agreed - that would be simpler. I was following the pattern employed by the `Index` and `List` components (which can probably change too). I guess this would only be useful if we intend to reuse the components elsewhere and change the loading message (which seems unlikely to me).